CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Support for Wazuh 4.5.2
 
+### Fixed
+
+- Fixed the rendering of 7 step into deployment of news agents that show a wrongs commands. [#5713](https://github.com/wazuh/wazuh-kibana-app/issues/5713)
```

Comment:
I would change this description. Something like 
"Fixed an error with the commands in the Deploy new agent section for Oracle Linux 6+ agents". Additionally, we should use the PR link instead of the issue's to be consistent with the rest of the file.

Reply:
Fixed in 3444d3b721a97c31f2930c4deb4e51399605f084